hack/apiserver.config

```diff
@@ -0,0 +1,27 @@
+/registry/actions/,tenant_begin,tenant_end,default

```

Comment:
Can we just hardcode default tenant as always watch in the code? This config seems involving more complicated code and not generic

Reply:
Removed